Intersect.Server.Core/Localization/Strings.cs

```diff
@@ -1097,6 +1100,9 @@ public sealed partial class PlayerNamespace : LocaleNamespace
         [JsonProperty(NullValueHandling = NullValueHandling.Ignore)]
         public readonly LocalizedString Left = @"{00} has left {01}.";
 
+        [JsonProperty(NullValueHandling = NullValueHandling.Ignore)]
+        public readonly LocalizedString LevelDown = @"You have leveled down! You are now level {00}!";
```

Comment:
This isn't super natural in english, it would be better to say `You have lost a level! You are now level {00}!`

Reply:
Also change the name of the string to `LevelLost`